services/sidecar/docker/entrypoint.sh

```diff
@@ -25,56 +30,82 @@ then
     DEVEL_MOUNT=/devel/services/sidecar
 
     stat $DEVEL_MOUNT > /dev/null 2>&1 || \
-        (echo "$ERROR" "You must mount '$DEVEL_MOUNT' to deduce user and group ids" && exit 1) # FIXME: exit does not stop script
+        (echo "$ERROR" "You must mount '$DEVEL_MOUNT' to deduce user and group ids" && exit 1)
 
-    USERID=$(stat -c %u $DEVEL_MOUNT)
-    GROUPID=$(stat -c %g $DEVEL_MOUNT)
-    GROUPNAME=$(getent group "${GROUPID}" | cut -d: -f1)
+    USERID=$(stat --format=%u $DEVEL_MOUNT)
```

Comment:
Mapping SCU to the host userid does not happen ONLY in development! Check the if statement !